src/web/components/ArticleMeta.tsx

```diff
@@ -264,7 +265,10 @@ export const ArticleMeta = ({
 
 	const showAvatar = onlyOneContributor && shouldShowAvatar(format);
 	return (
-		<div css={metaContainer(format)}>
+		<div
+			className={interactiveLegacyClasses.metaContainer}
```

Comment:
I'm not sure how className and CSS work together when both specified - as the css compiles down into a classname itself.

Reply:
ah, when I tested the static class name was included as well as the auto-generated one, eg:

<img width="481" alt="Screenshot 2021-08-16 at 14 08 58" src="https://user-images.githubusercontent.com/45561419/129568836-b986a3a2-e92e-41be-85c9-a1ba4d55adb2.png">

does that answer your question?

Reply:
yes - thanks!